### PR TITLE
add cache control header in http response

### PR DESCRIPTION
**Pull request by dengmingtong**
Issue #, if available:

Description of changes:
Add Cache-Control header into response.

Checklist:

- [yes ] Have you successfully deployed to an AWS account with your changes?
- [yes] Have you successfully deployed to an AWS China account with your changes?
- [yes] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/solution/infrastructure/config/infra/cloudformation.yml [hunk 1]
```diff
@@ -202,10 +202,22 @@ Resources:
   # CloudFront
   # =============================================================================================
 
+  ResponseHeadersPolicy:
+    Type: AWS::CloudFront::ResponseHeadersPolicy
```
Comment (zxkane) [inline main/solution/infrastructure/config/infra/cloudformation.yml:206]: Is it supported for China regions?

File: main/solution/infrastructure/config/infra/cloudformation.yml [hunk 2]
```diff
@@ -202,10 +202,22 @@ Resources:
...
+    Properties:
+      ResponseHeadersPolicyConfig:
+        Name: Cache-Control-Response-Headers-Policy
+        Comment: Add Cache-Control header to responses
+        CustomHeadersConfig:
+          Items:
+            - Header: Cache-Control
+              Value: no-store, no-cache
```
Comment (zxkane) [inline main/solution/infrastructure/config/infra/cloudformation.yml:214]: Add `Cache-Control: no-cache`, it means if we don't cache the content in CDN?

Comment (dengmingtong) [reply main/solution/infrastructure/config/infra/cloudformation.yml:214]: It means client side does not cache the content, not CDN.

File: main/solution/infrastructure/config/infra/cloudformation.yml (hunk 1 above)
Comment (dengmingtong) [reply main/solution/infrastructure/config/infra/cloudformation.yml:206]: Yes, I have tested it on China regions.

**Review by zxkane — APPROVED** [review]

**Pull request merged**